File: src/java/src/quickfix/fix44/UserRequest.java
package quickfix.fix44;
import quickfix.FieldNotFound;
import quickfix.Group;
import quickfix.field.*;

public class UserRequest extends Message
{
  public UserRequest()
  {
    getHeader().setField(new MsgType("BE"));
  }
  public UserRequest(
    quickfix.field.UserRequestID aUserRequestID,
    quickfix.field.UserRequestType aUserRequestType,
    quickfix.field.Username aUsername ) {

    getHeader().setField(new MsgType("BE"));
    set(aUserRequestID);
    set(aUserRequestType);
    set(aUsername);
  }

  public void set(quickfix.field.UserRequestID value)
  { setField(value); }
  public quickfix.field.UserRequestID get(quickfix.field.UserRequestID  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.UserRequestID getUserRequestID() throws FieldNotFound
  { quickfix.field.UserRequestID value = new quickfix.field.UserRequestID();
    getField(value); return value; }
  public boolean isSet(quickfix.field.UserRequestID field)
  { return isSetField(field); }
  public boolean isSetUserRequestID()
  { return isSetField(923); }
  public void set(quickfix.field.UserRequestType value)
  { setField(value); }
  public quickfix.field.UserRequestType get(quickfix.field.UserRequestType  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.UserRequestType getUserRequestType() throws FieldNotFound
  { quickfix.field.UserRequestType value = new quickfix.field.UserRequestType();
    getField(value); return value; }
  public boolean isSet(quickfix.field.UserRequestType field)
  { return isSetField(field); }
  public boolean isSetUserRequestType()
  { return isSetField(924); }
  public void set(quickfix.field.Username value)
  { setField(value); }
  public quickfix.field.Username get(quickfix.field.Username  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.Username getUsername() throws FieldNotFound
  { quickfix.field.Username value = new quickfix.field.Username();
    getField(value); return value; }
  public boolean isSet(quickfix.field.Username field)
  { return isSetField(field); }
  public boolean isSetUsername()
  { return isSetField(553); }
  public void set(quickfix.field.Password value)
  { setField(value); }
  public quickfix.field.Password get(quickfix.field.Password  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.Password getPassword() throws FieldNotFound
  { quickfix.field.Password value = new quickfix.field.Password();
    getField(value); return value; }
  public boolean isSet(quickfix.field.Password field)
  { return isSetField(field); }
  public boolean isSetPassword()
  { return isSetField(554); }
  public void set(quickfix.field.NewPassword value)
  { setField(value); }
  public quickfix.field.NewPassword get(quickfix.field.NewPassword  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.NewPassword getNewPassword() throws FieldNotFound
  { quickfix.field.NewPassword value = new quickfix.field.NewPassword();
    getField(value); return value; }
  public boolean isSet(quickfix.field.NewPassword field)
  { return isSetField(field); }
  public boolean isSetNewPassword()
  { return isSetField(925); }
  public void set(quickfix.field.RawDataLength value)
  { setField(value); }
  public quickfix.field.RawDataLength get(quickfix.field.RawDataLength  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.RawDataLength getRawDataLength() throws FieldNotFound
  { quickfix.field.RawDataLength value = new quickfix.field.RawDataLength();
    getField(value); return value; }
  public boolean isSet(quickfix.field.RawDataLength field)
  { return isSetField(field); }
  public boolean isSetRawDataLength()
  { return isSetField(95); }
  public void set(quickfix.field.RawData value)
  { setField(value); }
  public quickfix.field.RawData get(quickfix.field.RawData  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.RawData getRawData() throws FieldNotFound
  { quickfix.field.RawData value = new quickfix.field.RawData();
    getField(value); return value; }
  public boolean isSet(quickfix.field.RawData field)
  { return isSetField(field); }
  public boolean isSetRawData()
  { return isSetField(96); }
}

File: src/.NET/fix50/UserRequest.cs
namespace QuickFix50
{

  public class UserRequest : Message
  {
    public UserRequest() : base(MsgType()) {}
    static QuickFix.MsgType MsgType() { return new QuickFix.MsgType("BE"); }

    public UserRequest(
      QuickFix.UserRequestID aUserRequestID,
      QuickFix.UserRequestType aUserRequestType,
      QuickFix.Username aUsername )
    : base(MsgType()) {
      set(aUserRequestID);
      set(aUserRequestType);
      set(aUsername);
    }

    public void set(QuickFix.UserRequestID value)
    { setField(value); }
    public QuickFix.UserRequestID get(QuickFix.UserRequestID  value)
    { getField(value); return value; }
    public QuickFix.UserRequestID getUserRequestID()
    { QuickFix.UserRequestID value = new QuickFix.UserRequestID();
      getField(value); return value; }
    public bool isSet(QuickFix.UserRequestID field)
    { return isSetField(field); }
    public bool isSetUserRequestID()
    { return isSetField(923); }

    public void set(QuickFix.UserRequestType value)
    { setField(value); }
    public QuickFix.UserRequestType get(QuickFix.UserRequestType  value)
    { getField(value); return value; }
    public QuickFix.UserRequestType getUserRequestType()
    { QuickFix.UserRequestType value = new QuickFix.UserRequestType();
      getField(value); return value; }
    public bool isSet(QuickFix.UserRequestType field)
    { return isSetField(field); }
    public bool isSetUserRequestType()
    { return isSetField(924); }

    public void set(QuickFix.Username value)
    { setField(value); }
    public QuickFix.Username get(QuickFix.Username  value)
    { getField(value); return value; }
    public QuickFix.Username getUsername()
    { QuickFix.Username value = new QuickFix.Username();
      getField(value); return value; }
    public bool isSet(QuickFix.Username field)
    { return isSetField(field); }
    public bool isSetUsername()
    { return isSetField(553); }

    public void set(QuickFix.Password value)
    { setField(value); }
    public QuickFix.Password get(QuickFix.Password  value)
    { getField(value); return value; }
    public QuickFix.Password getPassword()
    { QuickFix.Password value = new QuickFix.Password();
      getField(value); return value; }
    public bool isSet(QuickFix.Password field)
    { return isSetField(field); }
    public bool isSetPassword()
    { return isSetField(554); }

    public void set(QuickFix.NewPassword value)
    { setField(value); }
    public QuickFix.NewPassword get(QuickFix.NewPassword  value)
    { getField(value); return value; }
    public QuickFix.NewPassword getNewPassword()
    { QuickFix.NewPassword value = new QuickFix.NewPassword();
      getField(value); return value; }
    public bool isSet(QuickFix.NewPassword field)
    { return isSetField(field); }
    public bool isSetNewPassword()
    { return isSetField(925); }

    public void set(QuickFix.RawDataLength value)
    { setField(value); }
    public QuickFix.RawDataLength get(QuickFix.RawDataLength  value)
    { getField(value); return value; }
    public QuickFix.RawDataLength getRawDataLength()
    { QuickFix.RawDataLength value = new QuickFix.RawDataLength();
      getField(value); return value; }
    public bool isSet(QuickFix.RawDataLength field)
    { return isSetField(field); }
    public bool isSetRawDataLength()
    { return isSetField(95); }

    public void set(QuickFix.RawData value)
    { setField(value); }
    public QuickFix.RawData get(QuickFix.RawData  value)
    { getField(value); return value; }
    public QuickFix.RawData getRawData()
    { QuickFix.RawData value = new QuickFix.RawData();
      getField(value); return value; }
    public bool isSet(QuickFix.RawData field)
    { return isSetField(field); }
    public bool isSetRawData()
    { return isSetField(96); }

  };

}

